### PR TITLE
Made required changes in code for issue-3901

### PR DESCRIPTION
**Pull request by iaditichine**
I have made the required changes in code and updated readme.md file for issue-3901.

### CODE REVIEW

File: C-Plus-Plus/other/Sorting_an_array_of_zeroes_and_ones.cpp [hunk 1]
```diff
@@ -0,0 +1,74 @@
+/* Sorting an array of 0's and 1's
+
+Given an array of size n which contains only 0's and 1's in any order,we have to modify the array in such a way that 0's are on left side and 1's on the right side.
```
Comment (HarshCasper) [inline C-Plus-Plus/other/Sorting_an_array_of_zeroes_and_ones.cpp:3]: Modularize this piece of comment

File: C-Plus-Plus/other/Sorting_an_array_of_zeroes_and_ones.cpp [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+Test case-1:
+Input-1:
+N = 5
+arr[] = {0, 0, 1, 1, 0}
+Output-1: 0 0 0 1 1
+
+*/
+//CODE
```
Comment (HarshCasper) [inline C-Plus-Plus/other/Sorting_an_array_of_zeroes_and_ones.cpp:11]: Redundant Comment

**Review by HarshCasper — CHANGES_REQUESTED** [review]

**Comment by iaditichine** [comment]
How do I modularize this comment?

File: C-Plus-Plus/other/Sorting_an_array_of_zeroes_and_ones.cpp (hunk 1 above)
Comment (iaditichine) [reply C-Plus-Plus/other/Sorting_an_array_of_zeroes_and_ones.cpp:3]: How do I modularize this comment?

**Review by ankitaggarwal23 — COMMENTED** [review]
We don't accept PRs without any corresponding Issue. It is difficult for us to find your issue in tons of issues available. We can't review and merged incomplete PR's.  Closing this PR as of now. Open the new PR if you want this to be reviewed and add the proper details. Also do the changes as harsh mentioned in new PR.

**Pull request closed without merging**